docs/UserGuide.md

```diff
@@ -64,53 +66,230 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
+Deletes a specified event from a specified project.
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e. adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+Format: `deleteE PROJECT_INDEX r/EVENT_INDEX`
+
+* Deletes the event at the specified `EVENT_INDEX` from the project at the specified `PROJECT_INDEX`.
+* The `PROJECT_INDEX` refers to the number shown beside the project in the side menu.
+* The `EVENT_INDEX` refers to the number shown beside the event when viewing the project. 
+* Both indexes **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `deleteE 1 r/1`
+* `deleteE 2 r/2`
+
+#### Deleting a deadline from a project: `deleteD`
+
+Deletes a specified deadline from a specified project.
+
+Format: `deleteD PROJECT_INDEX r/DEADLINE_INDEX`
+
+* Deletes the deadline at the specified `DEADLINE_INDEX` from the project at the specified `PROJECT_INDEX`.
+* The `PROJECT_INDEX` refers to the number shown beside the project in the side menu.
+* The `DEADLINE_INDEX` refers to the number shown beside the deadline when viewing the project.
+* Both indexes **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+* `deleteD 1 r/1`
+* `deleteD 2 r/2`
+
+#### Deleting a todo from a project: `deleteT`
+
+Deletes a specified todo from a specified project.
+
+Format: `deleteT PROJECT_INDEX r/TODO_INDEX`
+
+* Deletes the todo at the specified `TODO_INDEX` from the project at the specified `PROJECT_INDEX`.
+* The `PROJECT_INDEX` refers to the number shown beside the project in the side menu.
+* The `TODO_INDEX` refers to the number shown beside the todo when viewing the project.
+* Both indexes **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+* `deleteT 1 r/1`
+* `deleteT 2 r/2`
+
+#### Deleting a contact from a project: `deleteCfrom`
+
+Deletes a specified contact from a specified project.
+
+Format: `deleteCfrom PROJECT_INDEX r/CONTACT_INDEX`
+
+* Deletes the contact at the specified `CONTACT_INDEX` from the project at the specified `PROJECT_INDEX`.
+* The `PROJECT_INDEX` refers to the number shown beside the project in the side menu.
+* The `CONTACT_INDEX` refers to the number shown beside the contact when viewing the project.
+* Both indexes **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+* `deleteCfrom 1 r/1`
+* `deleteCfrom 2 r/2`
+
+### Contacts Lists
```

Comment:
Should this be `Contact List` instead since we have only one such list?

Reply:
Agreed, thanks for being thorough.

Reply:
Changed it to contacts to be more consistent with the other level 3 heading  (projects).  Do you think this  is ok?

Reply:
Yes! That would be nice!